devops/aws/deploy-single-node.sample.cfg

```diff
@@ -20,4 +20,5 @@ CHAIN_SPEC_FILE="https://github.com/Joystream/joystream/releases/download/v9.14.
 # For deploy playground playbook only
 GIT_REPO="https://github.com/Joystream/joystream.git"
 BRANCH_NAME="master"
-SKIP_CHAIN_SETUP="false"
\ No newline at end of file
+SKIP_CHAIN_SETUP="false"
+ALL_PROPOSALS_PARAMETERS_JSON=""
```

Comment:
is there a difference between setting this to an empty string explicitly and not setting it at all in the bash script test `if [[ -z "$ALL_PROPOSALS_PARAMETERS_JSON" ]];` ?

Reply:
Kept this as an empty string to let the user know that they have the option to set this env variable, as they will refer the sample config while running the script. This doesn't effect the bash script. Let me know if it's not needed and I'll remove it.